add TaskListPage (page with list view) with test values

senkodima opened the pull request.

senkodima committed:
add TaskListPage (page with list view) with test values

senkodima committed:
change string[] to List<string> in TaskListViewModel

senkodima committed:
changing in User model (add TaskList Property)

a-mazurkevich commented on RegistrationApp/RegistrationApp/ViewModels/TaskListViewModel.cs:
```diff
@@ -0,0 +1,26 @@
+            set { SetField(ref _taskList, value); }
+        }
+
+        User user;
```
private User _user;

a-mazurkevich commented on RegistrationApp/RegistrationApp/ViewModels/TaskListViewModel.cs:
```diff
@@ -0,0 +1,26 @@
+
+        public TaskListViewModel()
+        {
+            user = new User()
```
Why do you create new 'User' if you know his pass and username when you redirected to TaskListPage

senkodima committed:
Create local database to save users and user tasks, add validation to email uniqueness

senkodima committed:
refactor code

senkodima committed:
Merge branch 'master' into #9-Create-local-DB-for-users-and-their-list-data

# Conflicts:
#	RegistrationApp/RegistrationApp/Autorization/AutorizationService.cs
#	RegistrationApp/RegistrationApp/Models/User.cs
#	RegistrationApp/RegistrationApp/RegistrationApp.csproj

senkodima committed:
refactor code

senkodima committed:
check user credentials from database

senkodima committed:
update nuget packages

senkodima committed:
Merge branch 'master' into #8-Create-Task-list-page

# Conflicts:
#	RegistrationApp/RegistrationApp/Models/User.cs
#	RegistrationApp/RegistrationApp/Pages/TaskListPage.xaml
#	RegistrationApp/RegistrationApp/Pages/TaskListPage.xaml.cs
#	RegistrationApp/RegistrationApp/ViewModels/SignInViewModel.cs
#	RegistrationApp/RegistrationApp/ViewModels/TaskListViewModel.cs

senkodima committed:
Merge branch '#9-Create-local-DB-for-users-and-their-list-data' into #8-Create-Task-list-page

# Conflicts:
#	RegistrationApp/RegistrationApp/Models/User.cs

senkodima committed:
refactor task list page
1) add button to add new random tasks lo listview
2) display all added tasks on listview
